hack/update_owners.py

```diff
@@ -73,28 +73,30 @@ def get_maintainers():
     # [].slice.call(document.querySelectorAll('.team-member-username a')).map(
     #     e => e.textContent.trim())
     ret = {"a-robinson", "alex-mohr", "amygdala", "andyzheng0831", "apelisse",
-        "aronchick", "ArtfulCoder", "bgrant0607", "bgrant0607-nocc",
-        "bprashanth", "brendandburns", "caesarxuchao", "childsb", "cjcullen",
-        "david-mcmahon", "davidopp", "dchen1107", "deads2k", "derekwaynecarr",
-        "dubstack", "eparis", "erictune", "fabioy", "fejta", "fgrzadkowski",
-        "freehan", "ghodss", "girishkalele", "gmarek", "goltermann",
-        "grodrigues3", "hurf", "ingvagabund", "ixdy",
-        "jackgr", "janetkuo", "jbeda", "jdef", "jingxu97", "jlowdermilk",
-        "jsafrane", "jszczepkowski", "justinsb", "kargakis", "karlkfi",
-        "kelseyhightower", "kevin-wangzefeng", "krousey", "lavalamp",
-        "liggitt", "luxas", "madhusudancs", "maisem", "mansoorj", "matchstick",
-        "mikedanese", "mml", "mtaufen", "mwielgus", "ncdc", "nikhiljindal",
-        "piosz", "pmorie", "pwittrock", "Q-Lee", "quinton-hoole", "Random-Liu",
-        "rmmh", "roberthbailey", "ronnielai", "saad-ali", "sarahnovotny",
-        "smarterclayton", "soltysh", "spxtr", "sttts", "swagiaal", "thockin",
-        "timothysc", "timstclair", "tmrts", "vishh", "vulpecula", "wojtek-t",
-        "xiang90", "yifan-gu", "yujuhong", "zmerlynn"}
+           "aronchick", "bgrant0607", "bgrant0607-nocc", "bprashanth",
+           "brendandburns", "caesarxuchao", "childsb", "cjcullen",
+           "david-mcmahon", "davidopp", "dchen1107", "deads2k",
+           "derekwaynecarr", "dubstack", "eparis", "erictune", "fabioy",
+           "fejta", "fgrzadkowski", "freehan", "ghodss", "girishkalele",
+           "gmarek", "goltermann", "grodrigues3", "hurf", "ingvagabund", "ixdy",
+           "jackgr", "janetkuo", "jbeda", "jdef", "jfrazelle", "jingxu97",
+           "jlowdermilk", "jsafrane", "jszczepkowski", "justinsb", "kargakis",
+           "karlkfi", "kelseyhightower", "kevin-wangzefeng", "krousey",
+           "lavalamp", "liggitt", "luxas", "madhusudancs", "maisem", "mansoorj",
+           "matchstick", "mbohlool", "mikedanese", "mml", "mtaufen", "mwielgus",
+           "ncdc", "nikhiljindal", "piosz", "pmorie", "pwittrock", "Q-Lee",
+           "quinton-hoole", "Random-Liu", "rmmh", "roberthbailey", "ronnielai",
+           "saad-ali", "sarahnovotny", "smarterclayton", "soltysh", "spxtr",
+           "sttts", "swagiaal", "thockin", "timothysc", "timstclair", "tmrts",
+           "vishh", "vulpecula", "wojtek-t", "xiang90", "yifan-gu", "yujuhong",
+           "zmerlynn"}
     return sorted(ret - SKIP_MAINTAINERS)
 
 
 def main():
     test_history = get_test_history()
     test_names = sorted(set(map(normalize, test_history['test_names'])))
+    test_names.append('$DEFAULT')
```

Comment:
I agree with @fejta that the $ is confusing.

also, the case doesn't match that PR - does this matter?